shapeshift/lib
# feat: thor btcquote

Thorchain getQuote fully working.

Can be tested locally linked to this web PR:
https://github.com/shapeshift/web/pull/2153

To Test:

- set REACT_APP_MIDGARD_URL in web .env to a known good url midgard (https://midgard.thorswap.net/v2 is pretty good)
- REACT_APP_FEATURE_THOR=true

Run web linked locally and see valid quotes & prices appearing for thorchain pairs (BTC/FOX for example)
